content/docs/tasks/traffic-management/secure-ingress-sds/index.md

```diff
@@ -0,0 +1,488 @@
+    $ pushd mtls-go-example
+    $ ./generate.sh httpbin.example.com <password>
+    $ mkdir ~+1/httpbin.new.example.com && mv 1_root 2_intermediate 3_application 4_client ~+1/httpbin.new.example.com
+    $ popd
+    $ kubectl create -n istio-system secret generic httpbin-credential --from-file=key=httpbin.new.example.com/3_application/private/httpbin.example.com.key.pem --from-file=cert=httpbin.new.example.com/3_application/certs/httpbin.example.com.cert.pem
+    {{< /text >}}
+
+1.  Access the `httpbin` service using _curl_
+
+    {{< text bash >}}
+    $ curl -v -HHost:httpbin.example.com --resolve httpbin.example.com:$SECURE_INGRESS_PORT:$INGRESS_HOST --cacert httpbin.new.example.com.b/2_intermediate/certs/ca-chain.cert.pem https://httpbin.example.com:$SECURE_INGRESS_PORT/status/418
+    ...
+    HTTP/2 418
+    ...
+    -=[ teapot ]=-
+
+       _...._
+     .'  _ _ `.
+    | ."` ^ `". _,
+    \_;`"---"`|//
+      |       ;/
+      \_     _/
+        `"""`
+    {{< /text >}}
+
+1.  Accessing `httpbin` with previous cert chain would fail
+
+    {{< text bash >}}
+    $ curl -v -HHost:httpbin.example.com --resolve httpbin.example.com:$SECURE_INGRESS_PORT:$INGRESS_HOST --cacert httpbin.example.com/2_intermediate/certs/ca-chain.cert.pem https://httpbin.example.com:$SECURE_INGRESS_PORT/status/418
+    ...
+    * TLSv1.2 (OUT), TLS handshake, Client hello (1):
+    * TLSv1.2 (IN), TLS handshake, Server hello (2):
+    * TLSv1.2 (IN), TLS handshake, Certificate (11):
+    * TLSv1.2 (OUT), TLS alert, Server hello (2):
+    * SSL certificate problem: unable to get local issuer certificate
+    {{< /text >}}
+
+## Configure a TLS ingress gateway with secret discovery service enabled for multiple hosts
+
+In this section you will configure an ingress gateway for multiple hosts, `httpbin.example.com` and `helloworld-v1.example.com`. The ingress gateway will retrieve unique credentials corresponding to specific `credentialName`.
+
+1.  Start the `helloworld-v1` sample
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -f -
+    apiVersion: v1
+    kind: Service
+    metadata:
+      name: helloworld-v1
+      labels:
+        app: helloworld-v1
+    spec:
+      ports:
+      - name: http
+        port: 8000
+      selector:
+        app: helloworld-v1
+    ---
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    metadata:
+      name: helloworld-v1
+    spec:
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: helloworld-v1
+        spec:
+          containers:
+          - name: helloworld
+            image: istio/examples-helloworld-v1
+            resources:
+              requests:
+                cpu: "100m"
+            imagePullPolicy: IfNotPresent #Always
+            ports:
+            - containerPort: 5000
+    EOF
+    {{< /text >}}
+
+1.  Create secret for ingress gateway. Assume you already create secret `httpbin-credential`. Now create secret `helloworld-credential`.
```

Comment:
a secret

Reply:
Assuming you've already created the `httpbin-credential` secret, now create the `hellowoerld-credential` secret.

Reply:
Done.